CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [2.5.3] - 2024-10-09
+
+* Updated
+  * Match device_api_version to mix release version (#252)
```

Comment:
This also includes all the extension work, right? It probably warrants more comments and maybe even a minor bump?

Reply:
That's what I get for putting this together and talking to Digit at the same time 😆 Fixed.